tldw_Server_API/tests/Agent_Client_Protocol/test_mcp_adapter.py

```
@@ -0,0 +1,337 @@
+    adapter = MCPAdapter()
+    with pytest.raises(RuntimeError, match="Not connected"):
+        await adapter.send_prompt([{"role": "user", "content": "hello"}])
+
+
+# ---------------------------------------------------------------------------
+# cancel
+# ---------------------------------------------------------------------------
+
+
+@pytest.mark.asyncio
+async def test_mcp_adapter_cancel(mock_transport, event_callback, collected_events):
```

Ruff (0.15.6)

[warning] Unused function argument: `collected_events`

(ARG001)